### PR TITLE
feat: use dejavu-browser prefixCls for antd styles

### PR DESCRIPTION
**Pull request by ShahAnuj2610**

### CODE REVIEW

**Review by jyash97 — COMMENTED** [review]
Have a few questions:

1. Are we having the styles for only the components that we use in dejavu or all the components?
2. We only need to use `ConfigProvider` should only be used at the top level once. Why are we wrapping all the components ?

**Comment by ShahAnuj2610** [comment]
> Have a few questions:
> 
> 1. Are we having the styles for only the components that we use in dejavu or all the components?
> 2. We only need to use `ConfigProvider` should only be used at the top level once. Why are we wrapping all the components ?

styles would break. one example is we are using `ConnectApp` at `QueryExplorer`, now the styles would break since it would `dejavu-browser` as a prefix. It is easily replicable.
- remove `ConfigProvider` wrapper from `ConnectApp`
- open QueryExplorer, styles would break

**Comment by jyash97** [comment]
- Okay for that you can just export the wrapper component instead and use the DefaultComponent where we already have a wrapper.

> Are we having the styles for only the components that we use in dejavu or all the components?

What about this issue? The size of this file is `736kb`

**Comment by lakhansamani** [comment]
@jyash97 since we are exporting those components from browser main and using inside `dejavu-main` independently we might need to wrap them and export.

For CSS file size, @ShahAnuj2610 can you share what is the size after the build, if the minified file size is small, let's keep it to ignore any side effects.

File: packages/browser/src/index.js [hunk 1]
```diff
@@ -23,20 +24,51 @@ import * as constants from './constants';
...
-				<DataBrowserContainer hasCloneApp={props.hasCloneApp} />
-			</section>
-		</BrowserRouter>
-	</Provider>
+	<WithConfigProvider>
+		<Provider store={store}>
+			<BrowserRouter>
+				<section>
+					<FlashMessage />
+					<ConnectApp {...props} />
```
Comment (lakhansamani) [inline packages/browser/src/index.js]: @ShahAnuj2610 you can use `<DefaultConnectApp {...props} />` here. as you are already wrapping with config provider

File: packages/browser/src/index.js [hunk 2]
```diff
@@ -23,20 +24,51 @@ import * as constants from './constants';
...
-				{/* eslint-disable-next-line react/prop-types */}
-				<DataBrowserContainer hasCloneApp={props.hasCloneApp} />
-			</section>
-		</BrowserRouter>
-	</Provider>
+	<WithConfigProvider>
+		<Provider store={store}>
+			<BrowserRouter>
+				<section>
+					<FlashMessage />
```
Comment (lakhansamani) [inline packages/browser/src/index.js]: @ShahAnuj2610 you can use `<DefaultFlashMessage />` here

**Review by lakhansamani — CHANGES_REQUESTED** [review]

**Review by lakhansamani — APPROVED** [review]

**Pull request merged**